addon/helpers/capitalize.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+const {
+  Helper: { helper },
+  String
```

Comment:
I had tried that earlier, but after running `ember test` I got some `Duplication declaration "capitalize"` errors:

```
Build failed.
File: modules/ember-composable-helpers/helpers/capitalize.js
modules/ember-composable-helpers/helpers/capitalize.js: Line 7: Duplicate declaration "capitalize"
TypeError: modules/ember-composable-helpers/helpers/capitalize.js: Line 7: Duplicate declaration "capitalize"
   5 | } = Ember;
   6 |
>  7 | export function capitalize([string]) {
     |                 ^
   8 |   return capitalize(string);
   9 | }
  10 |
```
Any suggestions on how to destructure here?